app/file.go

```diff
@@ -621,6 +621,30 @@ func (a *App) GetFileInfo(fileId string) (*model.FileInfo, *model.AppError) {
 	return result.Data.(*model.FileInfo), nil
 }
 
+func (a *App) GetFileThumbnail(fileId string) ([]byte, *model.AppError) {
```

Comment:
@jwilander, is this method strictly useful in the API, vs. a helper method that uses the existing `GetFileInfo` and `ReadFile`? I know we're on a quest to expose all the `Client4` can do, but I'm wondering how useful it is in practice to get the thumbnail without the `FileInfo`. This doesn't block the review, but just want to clarify the end goal here :)

Reply:
Hmm yeah this might make more sense as a helper function if there are already existing APIs that allow for this functionality

Reply:
@lieut-data @jwilander Do I need to make any changes?